source/javascripts/lib/wirsing/utils/pager.coffee

```diff
@@ -0,0 +1,14 @@
+#= require ../module
+
+# creates a list that makes paginating a list simple
+
+wirsing.module 'utils.Pager', ->
+  class Pager
+    constructor: (@options={}) ->
+      @pageSize     = @options.perPage || 30
+      @list         = []
+      @visibleCount = @pageSize
+
+    setList: (newList) -> @list = newList
+    nextPage:    -> @visibleCount += @pageSize
+    currentList: -> @list.slice(0,@visibleCount)
```

Comment:
As a side-note, I have to agree with @pferdefleisch that it also took me some time to understand the filter API. OOP would have made these things much easier to grasp, but probably result in more / less flexible code.